### PR TITLE
Switch the logic in querying Kubernetes cluster config, and removed `<br />`

### PR DESCRIPTION
**Pull request by acald-creator**
Signed-off-by: Antonette Caldwell <pullmana8@gmail.com>

**Notes for Reviewers**

This PR switches the logic.

Anytime you use `mesheryctl system status`, it looks to see if you have the Kubernetes config on your local computer. However, the error message used for this situation was not correct.

I switch the name of the Error Code to the right place, and also removed `<br />` from the line.

Also added newline in the appropriate place, but this can be changed.

Here is before:
```bash
./mesheryctl system status
Error: stat /home/phxvlabs-io/.kube/config: no such file or directory. Your local context is configured to run Meshery on Kubernetes using the Default token.<br />The Kubernetes cluster is not accessible. Please confirm that the token is valid. See https://docs.meshery.io/installation/quick-start for additional instructions
```

Here is the output after the changed.

```bash
./mesheryctl system status
Error: stat /home/phxvlabs-io/.kube/config: no such file or directory. Your local context is configured to run Meshery on Kubernetes using the Default token.
The Kubernetes cluster is not accessible. Please confirm that the cluster is running. See https://docs.meshery.io/installation/quick-start for additional instructions.
```

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
Looks good to me. 👍

File: mesheryctl/internal/cli/root/system/error.go [hunk 1]
```diff
@@ -94,12 +94,12 @@ func ErrRestartMeshery(err error) error {
...
 }
 
-func ErrK8sConfig(err error) error {
-	return errors.New(ErrK8sConfigCode, errors.Alert, []string{"The Kubernetes cluster is not accessible."}, []string{err.Error(), "<br />The Kubernetes cluster is not accessible", " Please confirm that the token is valid", " See https://docs.meshery.io/installation/quick-start for additional instructions"}, []string{"Kubernetes cluster is unavailable and that the token is invalid"}, []string{"Please confirm that your cluster is available and that the token is valid. See https://docs.meshery.io/installation/quick-start for additional instructions"})
+func ErrK8SQuery(err error) error {
+	return errors.New(ErrK8sConfigCode, errors.Alert, []string{"The Kubernetes cluster is not accessible."}, []string{err.Error(), "\nThe Kubernetes cluster is not accessible", " Please confirm that the token is valid", " See https://docs.meshery.io/installation/quick-start for additional instructions"}, []string{"Kubernetes cluster is unavailable and that the token is invalid"}, []string{"Please confirm that your cluster is available and that the token is valid. See https://docs.meshery.io/installation/quick-start for additional instructions"})
```
Comment (hexxdump) [inline mesheryctl/internal/cli/root/system/error.go:102]: This should be ErrK8sQueryCode, isnt it  ?

Comment (acald-creator) [reply mesheryctl/internal/cli/root/system/error.go:102]: Good catch, I forgot to change that line.

**Comment by acald-creator** [comment]
// @leecalcote @hexxdump 

Okay done, forgot that line, I thought I had changed it, but missed the copy/paste

**Review by hexxdump — APPROVED** [review]
LGTM

**Review by alphaX86 — APPROVED** [review]
LGTM 👍

**Pull request merged**